### ISSUE TITLE
Threat Intelligence Feeds

### ISSUE THREAD

**Issue by scafroglia93** [Divested-Mobile/hypatia#63]
Enjoy

https://github.com/scafroglia93/blocklists

**Comment by SkewedZeppelin** [comment]
Hi!

Hypatia works on file hashes, not on domains.

But I have a project for that too! https://divested.dev/pages/dnsbl

And I already uses your lists :slightly_smiling_face:  https://github.com/divestedcg/Simple_Hosts_Merger/blob/master/blocklists-nc.txt#L135-L151

Seems you've added a license and more lists, I can move them over to the regular not-nc-only list. Thanks!

**Comment by scafroglia93** [comment]
I need to figure out if I can insert the hashes into the lists

**Comment by SkewedZeppelin** [comment]
This repo already tracks many of those blogs/sources: https://raw.githubusercontent.com/Neo23x0/signature-base/master/iocs/hash-iocs.txt

I have some more I'll push to a new repo soon

**Comment by SkewedZeppelin** [comment]
Latest combined list now includes all of your lists: https://github.com/divestedcg/Simple_Hosts_Merger/commit/af6a65e904a3f599638ffaae7f1204f09ded3feb

Thank you!

**Comment by scafroglia93** [comment]
Is it a problem if I use the lists with domains to insert IP and hash? 

 do you need a separate thing?

**Comment by SkewedZeppelin** [comment]
nah, your lists your rules: keep whatever works best/easiest for yourself first

my simple hosts merger already has regex for domain matching and should filter anything else out correctly, and the hypatia merger has hexadecimal hash matching regex too, so they should both work just fine and I can always adjust them if necessary

**Comment by scafroglia93** [comment]
I can proceed with entering the IP and hash without causing problems for other lists.

Is there a specific format for adding hash and ip ?

is this correct for you ?

https://github.com/scafroglia93/blocklists/commit/1fda4f78a7b622687b1631386787a4be9fe98cca

**Comment by SkewedZeppelin** [comment]
I think everyone just makes up their own format :)

What you have works just fine

**Comment by scafroglia93** [comment]
Microsoft Threat List Is ready the other in the next day

You can use it

**Comment by scafroglia93** [comment]
Let me know so I can proceed with the additions

**Comment by scafroglia93** [comment]
Let me know so I can proceed with the additions

telegram -> @scafroglia93

**Comment by SkewedZeppelin** [comment]
I've added them :rocket: 

Stats: https://divested.dev/MalwareScannerSignatures/
Processor update: https://github.com/Divested-Mobile/Hypatia/commit/f02e35523fcef29b03f52b8358a8863fb98d58e6

```
	blocklists-microsoft.txt
		md5: 0, sha1: 0, sha256: 15
	blocklists-qianxin.txt
		md5: 14, sha1: 0, sha256: 0
	blocklists-tag.txt
		md5: 0, sha1: 0, sha256: 9
```

**Comment by scafroglia93** [comment]
Nice let's do it

**Comment by scafroglia93** [comment]
It's work

I hope to be useful for the project

**Comment by SkewedZeppelin** [comment]
Thank you again @scafroglia93 :slightly_smiling_face:

**Comment by scafroglia93** [comment]
[securiteinfo]

**Comment by SkewedZeppelin** [comment]
@scafroglia93 I suspect those are released under a restrictive license and can't be used.

I emailed them a few years ago for permission to no response.

**Comment by scafroglia93** [comment]
You can use it without mention LOL

**Comment by scafroglia93** [comment]
Last questione -> it's possible download the signature for clamav ?

**Comment by SkewedZeppelin** [comment]
@scafroglia93 previously yes, but now no: the generated database is a serialized Guava bloom filter Java object processed on server side to reduce the download amount and speedup app loading

**Comment by scafroglia93** [comment]
I'm redoing the repo;  Can you reset the indicators you currently have?

**Comment by SkewedZeppelin** [comment]
I froze my copy to 526b3922ee0e59a99123829acbdb296386f66840

**Comment by scafroglia93** [comment]
You can use fresh source

I'm working on it now

It's better format that i found some missed entries related to the new assestment

**Comment by scafroglia93** [comment]
Source are now ok, you can reset and follow the actual git history

thanks
